tools/msftidy.rb

```diff
@@ -601,6 +601,18 @@ def check_invalid_url_scheme
     end
   end
 
+  # This checks looks for extranous commas
+  # Stuff with comments should match too: , #metasploit module ]
+  # but it should not match interpolation #{
+  def check_comma
+    test = @source.scan(/[^\n]+,(?:\s*#[^{][^\n]+)?\s*[\]\}]/)
+    unless test.empty?
+      test.each { |item|
+        info("Extra comma: #{item}")
```

Comment:
Perhaps "extraneous comma" would be more appropriate, and it'd match your comment above. :)